### PR TITLE
Add passkeys support back

### PR DESCRIPTION
**Pull request by kean**
To test:

- Verify that passkeys login works (see https://github.com/wordpress-mobile/WordPress-iOS/pull/22001)
- Verify that passkeys from third-party password managers, e.g. 1Password, work
- Verify that universal links for *.wordpress.com subdomains work
- (Regression Testing) Verify that Login with Apple works
- (Regression Testing) Verify that password autofill works

> [!IMPORTANT]  
> On my physical device, it required a device restart to start working

## Regression Notes
1. Potential unintended areas of impact: Passkeys, Universal Links
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

**Comment by enejb** [comment]
I was able to login to WordPress.com with an account that has passkey enabled 🥳 

Although the experience was also not working as expected some of the time. 

I can't say that I am able to replicate it thought. Consistently the error that I was seeing is 
File

The Universal link that I tested worked as expected for me taking me to the jetpack app/

**Comment by ehti** [comment]
Can we also pleasure ensure that if someone has an external provider (password manager) installed that supports passkeys (like 1Password), that option is also shown when using the security key feature? It was introduced in iOS 17. 

https://developer.apple.com/passkeys/

**Comment by kean** [comment]
Hey, @ehti. Thanks for the suggestion. It wasn't on my radar but I tested it with 1Password and it worked out of the box with no changes from my end.

File: WordPress/Jetpack/JetpackDebug.entitlements [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	<array>
 		<string>applinks:jetpack.com</string>
 		<string>webcredentials:wordpress.com</string>
+        <string>webcredentials:*.wordpress.com</string>
```
Comment (momo-ozawa) [inline WordPress/Jetpack/JetpackDebug.entitlements]: [nit] the indentation seems wrong

**Review by momo-ozawa — APPROVED** [review]
Works as described - verified third party passkeys also work. LGTM w 1 small nit. I haven't been able to reproduce the error Enej mentioned

File: WordPress/Jetpack/JetpackDebug.entitlements (hunk 1 above)
Comment (kean) [reply WordPress/Jetpack/JetpackDebug.entitlements]: Fixed!

**Comment by kean** [comment]
I identified a way to reproduce the WodPressKit error and opened an [issue](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/817). It doesn't seem to be related to this PR.

**Pull request merged**